platform/darwin/test/MGLMapViewTests.m

```diff
@@ -41,4 +41,104 @@ - (void)testCoordinateBoundsConversion {
     XCTAssertTrue(CGRectIntersectsRect(spanningBoundsRect, rightAntimeridianBoundsRect), @"Something");
 }
 
+- (void)testUserTrackingModeCompletion {
+    __block BOOL completed = NO;
+    [mapView setUserTrackingMode:MGLUserTrackingModeNone animated:NO completionHandler:^{
```

Comment:
Tests currently fail to build on CI because the macOS implementation of MGLMapView hasn’t been updated. User tracking mode and target coordinate aren’t supported on macOS, so these tests will need to be conditionally compiled out.